README.md

```diff
@@ -340,6 +341,28 @@ func isError(error: Error?) -> Bool {
 }
 ```
 
+### Implicit Getters ###
+
+When overriding only the getter of a property, omit the use of `get`:
```

Comment:
Added a small note about other cases. I don't think it's that useful, but it's there.